routes/index.js

```diff
@@ -13,6 +13,6 @@ router.use('/ping', pingRouter)
 // categories routes
 router.use('/categories', categoriesRouter)
 
-router.use('/news/:id', newsRouter)
+router.use('/news/', newsRouter)
```

Comment:
```suggestion
router.use('/news', newsRouter)
```
no se si afecta la ultima / pero por las dudas lo marco

Reply:
borrar esa "/" final

Reply:
Entiendo que no afecta pero igual la quito para seguir el mismo criterio que el resto.